tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -126,6 +126,14 @@ bool IsTensorRTCandidate(const tensorflow::Node* node) {
     "Prod",
     "Max",
     "Min",
+    "Relu6",
+#endif
+#if NV_TENSORRT_MAJOR >= 5
```

Comment:
What happen if user ask to do FP32/FP16 for a graph with these ops? I know this sounds stupid and normal people shouldn't ask for that, but to make the logic correct, I think the converter should skip handling them instead of removing them. Maybe we can add a precision_mode parameter in this `IsTensorRTCandidate()` function for that.

Reply:
+1